packages/backend/src/accounting/service.ts

```diff
@@ -81,13 +124,19 @@ export interface AccountingService {
   createWithdrawal(withdrawal: Withdrawal): Promise<void | TransferError>
   postWithdrawal(id: string): Promise<void | TransferError>
   voidWithdrawal(id: string): Promise<void | TransferError>
+  getAccountTransfers(
```

Comment:
TigerBeetle now supports fetching of transfers. Adding to the accounting service later.